src/main/kotlin/com/vaadin/gradle/VaadinBuildFrontendTask.kt

```diff
@@ -140,6 +148,13 @@ open class VaadinBuildFrontendTask : DefaultTask() {
             remove(Constants.FRONTEND_TOKEN)
             remove(Constants.SERVLET_PARAMETER_ENABLE_PNPM)
```

Comment:
@mvysny some properties have been deprecated like `Constants.SERVLET_PARAMETER_ENABLE_PNPM` and still used in vaadin-maven-plugin like that, 
Do you want that we resolve deprecation and use `InitParameters.SERVLET_PARAMETER_ENABLE_PNPM` intead or keep it as it is ?

Reply:
I try to follow the Vaadin Maven Plugin code as closely as possible, so that it's easy to port any changes done to the Maven Plugin. However, in this particular case I think there's no harm in using the non-deprecated `InitParameters.SERVLET_PARAMETER_ENABLE_PNPM` - please go ahead :+1:

Reply:
Done :+1: I also added a test that asserts that vaadinClean task removes the TS-related files.